content/blog/2017-06-21-prometheus-20-alpha3-new-rule-format.md

```diff
@@ -0,0 +1,71 @@
+---
+ title: Prometheus 2.0 Alpha.3 with New Rule Format
+ created_at: 2017-06-21
+ kind: article
+ author_name: Goutham Veeramachaneni
+---
+
+Today we release the third alpha version of Prometheus 2.0. Aside from a variety of bug fixes in the new storage layer, it contains a few planned breaking changes.
+ 
+First, we moved to a new flag library, which uses the more common double-dash `--` prefix for flags instead of the single dash Prometheus used so far. Deployments have to be adapted accordingly.
+Additionally, further flags were removed with this alpha. The full list since Prometheus 1 is:
+ 
+`web.telemetry-path`<br/>
+All `storage.remote.*` flags<br/>
+All `storage.local.*` flags<br/>
+`query.staleness-delta`<br/>
+`alertmanager.url`<br/>
+ 
+Alerting and recording rules are on of the critical features of Prometheus.. But they also come with a few design issues and missing features, namely:
+ 
+* All rules are evaluated in concurrently, which is actually Prometheus’ oldest open bug. This has a couple of issues, the obvious one being that the load spikes every eval interval if you have a lot of rules. The other being that rules that depend on each other might not be fed the correct data. For example:
```

Comment:
Instead of "correct" I would say they might be getting "outdated" or "lagging" data - sounds less dramatic and also more to the point :)